### PR TITLE
 Adding & updating some Malayalam channels

### PR DESCRIPTION
**Pull request by zcraber**

### CODE REVIEW

File: data/channels.csv [hunk 1]
```diff
@@ -667,6 +667,7 @@ AjaraTV.ge,Ajara TV,,,,GE,,,c/GE,kat,,FALSE,,,,https://ajaratv.ge/,https://i.img
 AJKTV.pk,AJK TV,,,Pakistan Television Corporation,PK,,,c/PK,eng,,FALSE,2004-02-05,,,,https://i.imgur.com/Zb9lawv.png
 Ajman.ae,Ajman,,,,AE,,,c/AE,ara,,FALSE,,,,,https://i.imgur.com/rDekCeE.png
 AjmanTV.ae,Ajman TV,,,,AE,,,c/AE,ara,,FALSE,,,,https://www.ajmantv.com/,https://i.imgur.com/hVNIwgE.png
+AJNetNews.in,AJ Net News,,,,IN,,,c/IN,mal,news,FALSE,,,,https://ajnetnews.com/,https://i.imgur.com/tqJqNCY.png
```
Comment (freearhey) [inline data/channels.csv:670]: Image size should not exceed 512x512: https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#channels

```suggestion
AJNetNews.in,AJ Net News,,,,IN,,,c/IN,mal,news,FALSE,,,,https://ajnetnews.com/,https://i.imgur.com/DT9VB1Q.png
```

File: data/channels.csv [hunk 2]
```diff
@@ -3809,6 +3811,7 @@ CDirect.fr,CDirect,,,,FR,,,c/FR,fra,,FALSE,,,,,https://i.imgur.com/Zis4TRh.png
 CDMInternacional.pr,CDM Internacional,,,,PR,,,c/PR,spa;eng,,FALSE,,,,,https://i.imgur.com/jOpammm.png
 CDN.do,CDN,,,,DO,,,c/DO,spa,,FALSE,,,,https://cdn.com.do/,https://i.imgur.com/eQyjIgf.png
 CDNDeportes.do,CDN Deportes,,,,DO,,,c/DO,spa,,FALSE,,,,https://cdndeportes.com.do/,https://i.imgur.com/yU5LqTL.png
+CDNews.in,CD News,,,,IN,,,c/IN,mal,news,FALSE,,,,,https://i.imgur.com/VIPPsxf.png
```
Comment (freearhey) [inline data/channels.csv:3814]:
```suggestion
CDNews.in,CD News,,,,IN,,,c/IN,mal,news,FALSE,,,,,https://i.imgur.com/zIXsuC2.png
```
Removed the transparent background around the logo.

File: data/channels.csv [hunk 3]
```diff
@@ -4782,6 +4785,7 @@ ClubTV.cl,Club TV,,,,CL,CL-BI,Santa Juana,c/CL,spa,,FALSE,,,,https://www.clubtv.
 CMAC1.us,CMAC 1,,,,US,US-CA,Fresno,s/US-CA,eng,general,FALSE,,,,https://cmac.tv/public/,https://i.imgur.com/PYhdQy9.png
 CMAC2.us,CMAC 2,,,,US,US-CA,Fresno,s/US-CA,eng,education,FALSE,,,,https://cmac.tv/edu/,https://i.imgur.com/O0sYOED.png
 CMAC3.us,CMAC 3,,,,US,US-CA,Fresno,s/US-CA,eng,legislative,FALSE,,,,https://cmac.tv/gov/,https://i.imgur.com/ToZzguz.png
+CMalayalamTV.ca,C Malayalam TV,,,,CA,,,c/CA,mal,general,FALSE,,,,https://www.canadianmalayalamtv.com/,https://i.imgur.com/WPksZUC.png
```
Comment (freearhey) [inline data/channels.csv:4788]:
```suggestion
CMalayalamTV.ca,C Malayalam TV,,,,CA,,,c/CA,mal,general,FALSE,,,,https://www.canadianmalayalamtv.com/,https://i.imgur.com/rpg0bQA.png
```
Removed the transparent background around the logo.

File: data/channels.csv [hunk 4]
```diff
@@ -5906,6 +5910,7 @@ DobleCTelevision.pe,Doble C Television,Doble C Televisión,,,PE,PE-JUN,Mazamari,
 DocBox.mn,Doc Box,,,,MN,,,c/MN,mon,,FALSE,,,,,https://i.imgur.com/ziZpmv0.png
 DoceMisiones.ar,Doce Misiones,,,,AR,,,c/AR,spa,,FALSE,,,,http://www.canal12misiones.com/,https://i.imgur.com/agVTEeO.png
 Doctor.ru,Doctor,Доктор,,,RU,,,c/RU,rus,education;lifestyle,FALSE,,,,https://doc-tv.ru/,https://i.imgur.com/VNirxxn.png
+DoctorLive.in,Doctor Live,,,,IN,,,c/IN,mal,lifestyle,FALSE,,,,https://doctorlivemedia.in/,https://i.imgur.com/ZKGWABt.png
```
Comment (freearhey) [inline data/channels.csv:5913]:
```suggestion
DoctorLive.in,Doctor Live,,,,IN,,,c/IN,mal,lifestyle,FALSE,,,,https://doctorlivemedia.in/,https://i.imgur.com/D6cMaVs.png
```
Removed the transparent background around the logo.

File: data/channels.csv [hunk 5]
```diff
@@ -7896,6 +7903,7 @@ GospelLifeTV.za,GospelLife TV,,,,ZA,,,c/ZA,eng,religious,FALSE,,,,https://www.af
 GospelMovieTV.br,Gospel Movie TV,,,,BR,,,c/BR,por,religious,FALSE,,,,https://ipgotv.net.br/tv/gmovies/,https://i.imgur.com/cQN3nWt.png
 GospelTruthTV.us,Gospel Truth TV,,,,US,,,c/US,eng,religious,FALSE,,,,,https://i.imgur.com/LI52X2q.png
 GospelTV.et,Gospel TV,,,,ET,,,c/ET,amh,religious,FALSE,,,,,https://i.imgur.com/WzVncd0.png
+GosreeVision.in,GosreeVision,,,,IN,,,c/IN,mal,general,FALSE,,,,http://www.gosreetv.in/,https://i.imgur.com/XBWaCRm.png
```
Comment (freearhey) [inline data/channels.csv:7906]:
```suggestion
GosreeVision.in,GosreeVision,,,,IN,,,c/IN,mal,general,FALSE,,,,http://www.gosreetv.in/,https://i.imgur.com/pdBo8Xb.png
```
Removed the transparent background around the logo.

File: data/channels.csv [hunk 6]
```diff
@@ -7916,6 +7924,7 @@ GoWild.us,Go Wild,,,,US,,,c/US,eng,,FALSE,,,,https://watch.plex.tv/live-tv/chann
 GPRTV.id,GPR TV,,,,ID,,,c/ID,ind,,FALSE,2018-12-10,,,https://gprtv.id/,https://i.imgur.com/mYHiCJu.png
 GQ.us,GQ,,,,US,,,c/US,eng,lifestyle,FALSE,,,,,https://i.imgur.com/mTiaf9I.png
 GR8TV.pk,GR 8 TV,,,,PK,,,c/PK,eng,,FALSE,,,,,https://i.imgur.com/wvJDwzL.png
+GraceFamilyTV.in,Grace Family TV,,,,IN,,,c/IN,mal,religious,FALSE,,,,https://www.gracefamilytv.com/,https://i.imgur.com/0mdXEeZ.png
```
Comment (freearhey) [inline data/channels.csv:7927]:
```suggestion
GraceFamilyTV.in,Grace Family TV,,,,IN,,,c/IN,mal,religious,FALSE,,,,https://www.gracefamilytv.com/,https://i.imgur.com/srOoujm.png
```
Removed the transparent background around the logo.

File: data/channels.csv [hunk 7]
```diff
@@ -10041,6 +10050,7 @@ KALBDT3.us,KALB-DT3,,CW,,US,US-LA,Alexandra,s/US-LA,eng,,FALSE,,,,,https://uploa
 KaleAwadiTV.et,Kale Awadi TV,,,,ET,,,c/ET,amh,,FALSE,,,,,https://i.imgur.com/reqk3BY.png
 KaleidoscopeTV.ru,Kaleidoscope TV,Калейдоскоп ТВ,,,RU,,,c/RU,rus,,FALSE,,,,http://www.kaleidoskoptv.tv/,https://i.imgur.com/Ub3gojM.png
 KalemehTV.uk,Kalemeh TV,,,,UK,,,c/IR,fas,religious,FALSE,,,,https://kalemeh.tv/,https://i.imgur.com/RsKKULI.png
+KalikaTV.in,Kalika TV,,,,IN,,,c/IN,mal,music,FALSE,,,,,https://i.imgur.com/KEZ5M4z.png
```
Comment (freearhey) [inline data/channels.csv:10053]:
```suggestion
KalikaTV.in,Kalika TV,,,,IN,,,c/IN,mal,music,FALSE,,,,,https://i.imgur.com/XWXFoJA.png
```
Removed the transparent background around the logo.

File: data/channels.csv [hunk 8]
```diff
@@ -20545,6 +20557,7 @@ SABCEducation.za,SABC Education,,SABC,South African Broadcasting Corporation,ZA,
 SABCNews.za,SABC News,,,,ZA,,,c/ZA,eng,news,FALSE,2013-01-01,,,http://www.sabc.co.za/,https://i.imgur.com/liLta8j.png
 SABCSport.za,SABC Sport,,SABC,South African Broadcasting Corporation,ZA,,,c/ZA,eng,sports,FALSE,,,,https://www.sabcsport.co.za/,https://upload.wikimedia.org/wikipedia/commons/3/38/SABC%2BSport%2B300x169.jpg
 SaberMas.us,Saber Mas,,,,US,,,c/US,spa,,FALSE,,,,https://watch.plex.tv/live-tv/channel/saber-mas,https://i.imgur.com/EfyaOXz.png
+SabhaTV.in,Sabha TV,,,,IN,,,c/IN,mal,legislative,FALSE,2020-08-17,,,https://sabhatv.com/,https://i.ibb.co/4NcM4hV/512px-Sabha-TV-Logo-svg.png
```
Comment (freearhey) [inline data/channels.csv:20560]:
```suggestion
SabhaTV.in,Sabha TV,,,,IN,,,c/IN,mal,legislative,FALSE,2020-08-17,,,https://sabhatv.com/,https://i.imgur.com/iaegMMm.png
```
Removed the transparent background around the logo.

File: data/channels.csv [hunk 9]
```diff
@@ -21986,6 +22000,7 @@ StarMoviesIndia.in,Star Movies India,,,,IN,,,c/IN,hin,movies,FALSE,2011-04-01,,,
 StarMoviesMiddleEast.ae,Star Movies Middle East,,,,AE,,,r/MIDEAST,ara,movies,FALSE,,,,http://foxarabia.tv/,https://i.imgur.com/JhT3k6R.png
 StarMoviesPlus.tz,Star Movies +,,,,TZ,,,c/TZ,eng,movies,FALSE,,,,,https://i.imgur.com/ljxL3O6.png
 StarMoviesSelect.in,Star Movies Select,,,,IN,,,c/IN,hin,movies,FALSE,2015-07-09,,,http://www.starmovies.in/,https://i.imgur.com/UomTc3O.png
+Starnet.in,Starnet,,,,IN,,,c/IN,mal,general,FALSE,,,,,https://i.imgur.com/bYqsCr1.png
```
Comment (freearhey) [inline data/channels.csv:22003]:
```suggestion
Starnet.in,Starnet,,,,IN,,,c/IN,mal,general,FALSE,,,,,https://i.imgur.com/J3jB7Kr.png
```
Removed the transparent background around the logo.

File: data/channels.csv [hunk 10]
```diff
@@ -25685,12 +25700,14 @@ VatiLao.la,Vati Lao,,,,LA,,,c/LA,lao,entertainment;shop,FALSE,,,,,https://i.imgu
 Vavoom.rs,Vavoom,,,,RS,,,c/RS,srp,,FALSE,,,,,https://i.imgur.com/d9s1See.png
 VavTV.tr,Vav TV,,,,TR,,,c/TR,tur,,FALSE,,,,https://www.vavtv.com.tr/,https://i.imgur.com/jw0gB8L.png
 VayasemanitaSpain.es,Vaya semanita Spain,,,,ES,,,c/ES,spa,,FALSE,,,,https://pluto.tv/en/live-tv/vaya-semanita-es,https://i.imgur.com/0bCLNUG.png
+VBCNews.in,VBC News,,,,IN,,,c/IN,mal,news,FALSE,,,,https://vbcnewsthodupuzha.com/,https://i.imgur.com/tdIewPd.png
```
Comment (freearhey) [inline data/channels.csv:25703]:
```suggestion
VBCNews.in,VBC News,,,,IN,,,c/IN,mal,news,FALSE,,,,https://vbcnewsthodupuzha.com/,https://i.imgur.com/SvXCjha.png
```
Removed the transparent background around the logo.

File: data/channels.csv [hunk 11]
```diff
@@ -25685,12 +25700,14 @@ VatiLao.la,Vati Lao,,,,LA,,,c/LA,lao,entertainment;shop,FALSE,,,,,https://i.imgu
...
 VavTV.tr,Vav TV,,,,TR,,,c/TR,tur,,FALSE,,,,https://www.vavtv.com.tr/,https://i.imgur.com/jw0gB8L.png
 VayasemanitaSpain.es,Vaya semanita Spain,,,,ES,,,c/ES,spa,,FALSE,,,,https://pluto.tv/en/live-tv/vaya-semanita-es,https://i.imgur.com/0bCLNUG.png
+VBCNews.in,VBC News,,,,IN,,,c/IN,mal,news,FALSE,,,,https://vbcnewsthodupuzha.com/,https://i.imgur.com/tdIewPd.png
 VBMediaTV.mx,VB Media TV,,,,MX,,Santiago de Querétaro,c/MX,spa,general,FALSE,,,,https://www.vbmedia.com.mx/,https://i.imgur.com/0W8kHxt.png
 VBS.ca,VBS,,,,CA,,,c/CA,vie,,FALSE,,,,,https://i.imgur.com/IWpevBp.jpg
 VBSTV.us,VBS TV,,,,US,,,c/US,vie,,FALSE,,,,,https://i.imgur.com/IWpevBp.jpg
 VCAT.us,VCAT,,,,US,US-CA,Vallejo,s/US-CA,eng,general,FALSE,,,,https://vcat.tv/,https://i.imgur.com/OrRSj8q.png
 VClassicTV.cl,V Classic TV,,,,CL,CL-CO,Coquimbo,c/CL,spa,music,FALSE,,,,https://www.radiocoquimbo.cl/,https://i.imgur.com/WIACJSy.png
 VCOnline.cl,VC Online,,,,CL,,,c/CL,spa,,FALSE,,,,https://vencio.wixsite.com/vcmedios,https://i.imgur.com/VlKl2VB.png
+VCV.in,VCV,,,,IN,,,c/IN,mal,general,FALSE,,,,,https://i.imgur.com/1cyXSJB.jpg
```
Comment (freearhey) [inline data/channels.csv:25710]:
```suggestion
VCV.in,VCV,,,,IN,,,c/IN,mal,general,FALSE,,,,,https://i.imgur.com/KbU36xI.png
```
Reduced the image size to 512px.

**Review by freearhey — CHANGES_REQUESTED** [review]

File: data/channels.csv (hunk 2 above)
Comment (zcraber) [reply data/channels.csv:3814]: It already had transparency!

File: data/channels.csv (hunk 3 above)
Comment (zcraber) [reply data/channels.csv:4788]: It already had transparency!

File: data/channels.csv (hunk 4 above)
Comment (zcraber) [reply data/channels.csv:5913]: It already had transparency!

File: data/channels.csv (hunk 7 above)
Comment (zcraber) [reply data/channels.csv:10053]: Already have a transparent background!

File: data/channels.csv (hunk 8 above)
Comment (zcraber) [reply data/channels.csv:20560]: The original image already has a transparent background!

File: data/channels.csv (hunk 9 above)
Comment (zcraber) [reply data/channels.csv:22003]: The original image already has a transparent background!

File: data/channels.csv (hunk 10 above)
Comment (zcraber) [reply data/channels.csv:25703]: The original image already has a transparent background!

**Comment by zcraber** [comment]
@freearhey The logos already had a transparent background.

**Comment by freearhey** [comment]
> @freearhey The logos already had a transparent background.

I'm aware of that. I only removed the transparent background AROUND the logo. 

Example:

image

**Comment by zcraber** [comment]
Oh ok, I was confused.

**Comment by zcraber** [comment]
I've committed the suggestions.

**Review by freearhey — CHANGES_REQUESTED** [review]
Most of the links have not been updated. Please check again.

**Comment by zcraber** [comment]
Something is wrong.
When I click `Commit suggestion`, I'm getting this error message: `This diff has recently been updated. Refresh and try again.`
image

**Review by freearhey — APPROVED** [review]

**Pull request merged**